src/main/java/com/github/jamesnetherton/zulip/client/api/message/Message.java

```diff
@@ -170,4 +173,12 @@ public List<String> getTopicLinks() {
     public MessageType getType() {
         return type;
     }
+    
+    public Long getStreamId() {
+        return streamId;
+	}
+    
+    public void setStreamId(Long streamId) {
+		this.streamId = streamId;
+	}
 }
```

Comment:
Minor formatting suggestion:

```suggestion
    public Long getStreamId() {
        return streamId;
    }

    public void setStreamId(Long streamId) {
        this.streamId = streamId;
    }
```